### ISSUE TITLE
OCR Anomalies

### ISSUE THREAD

**Issue by NoYBMe** [SubtitleEdit/subtitleedit#3749]
I wrote this as a supplemental report a couple of days ago on my previous OCR bug report. However, since it received no reply, I deleted that supplemental report and are reporting it here now as a new issue.

I was testing further on the previous OCR bug I reported and found a couple of anomalies worth mentioning;

1. OCR with Tesseract 302/410 is automatically translating [♪♪♪] as [NI] from an idx/sup.

The settings when this happens are:

Fix OCR errors CHECKED.
Prompt for unknown words CHECKED.
Try to guess unknown words UNCHECKED.
Auto break paragraph if more than two lines CHECKED.

This automatic OCR change does NOT prompt or show under:

Unknown words
All Fixes
Guesses used

It just inserts [NI] in place of [♪♪♪].

I don't have a screen-cap for this, but I can send you the idx/sup file (5MB) if you want.

2. OCR with Tesseract 302/410 and same settings as above will randomly and rarely skip the first line of a two line subtitle.

I think, I read about this before. I tried quick searching Google, Github, VideoHelp, Doom9 but could not find anything. Therefore, I'm bringing it to your attention.

For example:

- Go.

- Thanks.

Will OCR:

- Thanks.

Screen-cap:

![image](https://user-images.githubusercontent.com/54785488/64402399-a9a65e80-d029-11e9-8271-74864791411c.png)

**Comment by niksedk** [comment]
@NoYBMe: Please do send the file so I can take a look.
I would guess that these issues are related to the Tesseract engine and not SE... you can run the cmd line like this: https://github.com/tesseract-ocr/tesseract/wiki/Command-Line-Usage

**Comment by NoYBMe** [comment]
Attached find subtitles.zip containing the idx/sup/srt files.

You will find two sets of subtitles named subtitle1 and subtitle2, including the SE OCR converted SRT files.

The subtitle1 files are for the [♪♪♪] thing and the subtitle2 files are for the first line skip thing.

The [NI] in the SRT file starts showing up on line 2 and continue from there.

The first line skip thing shows up in the SRT file on line 300.

I will try the Tesseract command line when I get a chance and see. Until then, since you know your program better than I do, I will take your word/guess for it.

Thanks for the reply.

Attached-Subs:

[subtitles.zip](https://github.com/SubtitleEdit/subtitleedit/files/3587540/subtitles.zip)

**Comment by NoYBMe** [comment]
@niksedk:

I ran Tesseract 302/410 supplied with SE from CLI, processed the two respective images accordingly and the results were the same errors, therefore it seems the Tesseract version in use with SE are the cause and not SE itself.

I did not have time to download a copy of Tesseract source and compile it. So obviously I don't know if the results from that would be the same or if the errors have to do with the compiling instructions, or other variable, etc. I will check it out when I have more time and let you know if different results are found.

Take care and thanks.
